CONTRIBUTING.md

```diff
@@ -51,21 +51,18 @@ In a PR:
 
 1. Bump the version in the `package.json` file.
 2. Run the `npm update` command to update the `package-lock.json` file.
-3. Run the `npm run webpack` command to ensure the build system produces the latest distributable files.
-4. Search and replace every occurences of the previous version in the `README.md` file with the new version.
-5. Search and replace every occurences of the previous version in the `/examples` directory with the new version.
+3. Search and replace every occurrence of the previous version in the `README.md` file with the new version.
+4. Search and replace every occurrence of the previous version in the `/examples` directory with the new version.
```

Comment:
Now, code in `examples` directory doesn't work as-is, it requires running `npm run webpack` first.
Do you think that's fine? Or should we update the imports there to point to: `https://jslib.k6.io/aws`, and update the version in the import on each new release, as stated here?

Reply:
I think we should instead mention the `npm run webpack` step, as I would prefer examples to work "locally" for each commit.

Reply:
Okay, let me try to slightly modify docs to make such thing clear 👌🏻